Week-2/Assignment-1/line-up.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Line Up</title>
+    <link rel="stylesheet" href="line-up.css">
+
+  </head>
+  <body>
+    <main>
+      <h2>Starting Lineup</h2>
+      <form>
+        <fieldset>
+          <legend>Status:</legend>
+          <p>
+            <label>
+              <input type="radio" name="status" value="draft" /> Draft
+            </label>
+            <label>
+              <input type="radio" name="status" value="final" checked /> Final
+            </label>
+          </p>
+        </fieldset>
+
+        <fieldset>
+          <legend>Game Date:</legend>
+          <p>
+            <select name="gameDate">
+              <option value="March 20">March 20</option>
+              <option value="March 24">March 24</option>
+              <option value="April 2">April 2</option>
+              <option value="April 6">April 6</option>
+              <option value="April 13">April 13</option>
+            </select>
+          </p>
+        </fieldset>
```

Comment:
Here, we don't need to wrap the `Game Date` component inside `<fieldset>...</fieldset>`